script.js

```diff
@@ -92,16 +120,19 @@
 
 
 
+/*
+var x=Math.random().toString(36).substring(6);
+alert("This is your randomly generated alphanumeric string "+ x);
+*/
 
+/*
+var y=prompt("Enter the ength of a random alphanumeric string");
+var length= parseInt(y);
+var z=Math.random().toString(36).substring(length);
+alert(z);
```

Comment:
Good trick to convert a random float into a base 36 string. But the length of the substrings returned are incorrect...